docs/release-notes/v11.4.0.md

```diff
@@ -0,0 +1,138 @@
+---
+id: v11.4.0
+title: Version v11.4.0 Release Notes
+sidebar_label: v11.4.0
+image: "img/release-note-link-preview.png"
+---
+
+_July 06th, 2023_
+
+## Console
+
+### New Features
+
+#### What's new section
+
+A big welcome to the [What's new section](/development_suite/overview-dev-suite.md#whats-new), where you can stay up to date with the latest product news!  
+You can access this section by clicking on the question mark icon located in the top bar of the Console interface.
+In case there is any unread news, a notification symbol will appear, so that you do not miss the latest updates that can improve your experience on Mia-Platform Console!
+
+#### Support to automatic logout from identity provider
+
+From now on, Self-Hosted Mia-Platform Console installations support the configuration of user automatic logout from an OIDC compliant identity provider when logging out from the Console itself. For more information, visit the related [Self-Hosted installations documentation page](/self_hosted/installation-chart/30_authentication_provider.md#logout-flow).
```

Comment:
```suggestion
From now on, Mia-Platform Console installations support the configuration of user automatic logout from an OIDC compliant identity provider when logging out from the Console itself. For more information, visit the related [Mia-Platform installations documentation page](/self_hosted/installation-chart/30_authentication_provider.md#logout-flow).
```

Reply:
should we remove also the link to the self hosted installation documentation page?